lms/static/js/verify_student/views/step_view.js

```diff
@@ -20,9 +22,9 @@
 
             /* Mix non-conflicting functions from underscore.string
              * (all but include, contains, and reverse) into the
-             * Underscore namespace
+             * Underscore namespace.
              */
-            _.mixin( _s.exports() );
+             _.mixin(_.str.exports());
```

Comment:
I wrote my comment above before seeing this one. I definitely agree that we should move away from it in the future, and I also think that we should try to remove the current pattern too.